gnmi_server/server_test.go

```diff
@@ -107,6 +107,25 @@ func createServer(t *testing.T, port int64) *Server {
 	return s
 }
 
+func createReadServer(t *testing.T, port int64) *Server {
+	certificate, err := testcert.NewCert()
```

Comment:
As this is a helper function it should be marked as such by calling `t.Helper()` at its beginning.